test/karma.config.js

```diff
@@ -42,16 +42,21 @@ module.exports = function(config) {
         debug: true,
         logLevel: config.LOG_INFO,
         browserConsoleLogOptions,
+        browserDisconnectTimeout: 10000,
+        browserDisconnectTolerance: 1,
+        captureTimeout: 120000,
+        processKillTimeout: 100000,
         client: {
             captureConsole,
             mocha: {
-                timeout: 5000 // 5 seconds.  Increase from default 2 seconds.
+                timeout: 10000 // 10 seconds.  Increase from default 2 seconds.
```

Comment:
Strange, still seeing a 2000 ms issue here for firefox

<img width="1086" height="124" alt="Image" src="https://github.com/user-attachments/assets/debcb91b-cf07-4d91-ac1b-bf135eaad44f" />